extensions/dependencies/logic_proof.html

```diff
@@ -1,7 +1 @@
-<script src="/extensions/interactions/LogicProof/static/js/shared.js"></script>
-<script src="/extensions/interactions/LogicProof/static/js/student.js"></script>
-<script src="/extensions/interactions/LogicProof/static/js/teacher.js"></script>
-<script src="/extensions/interactions/LogicProof/static/js/data.js"></script>
-<script src="/extensions/interactions/LogicProof/static/js/generatedDefaultData.js"></script>
-<script src="/extensions/interactions/LogicProof/static/js/generatedParser.js"></script>
-<script src="/extensions/interactions/LogicProof/static/js/conversion.js"></script>
+
```

Comment:
Why do we still need this file? Shouldn't we delete it (and anything related to its existence)?

Reply:
Done